aim/sdk/repo.py

```diff
@@ -54,15 +55,21 @@ def __init__(self, path: str, *, read_only: bool = None, init: bool = False):
         if read_only is not None:
             raise NotImplementedError
         self.read_only = read_only
-        self.root_path = path
-        self.path = os.path.join(path, AIM_REPO_NAME)
+        self._mount_root = None
+        if path.startswith('ssh://'):
+            self._mount_root, self.root_path = mount_remote_repo(path)
+        else:
+            self.root_path = path
+        self.path = os.path.join(self.root_path, AIM_REPO_NAME)
 
         if init:
             os.makedirs(self.path, exist_ok=True)
```

Comment:
does this work for remote fs path?
I think it's better to keep `sefl.path` as the mounted local path, and keep `self._remote_path` when user provides `ssh://` path

Reply:
we won't need the remote path in any point of the execution, I had `_remote_path` in the initial version, but then decided not to keep it, and just use `_mount_root` for the purpose of checking if we have a remote repo mounted or not, and  `os.makedirs` operation works fine, because `self.root_path` will be already mounted to some local directory

Reply:
OK, got it. Thanks!